pyanaconda/payload/dnfpayload.py

```diff
@@ -327,11 +327,13 @@ def _replace_vars(self, url):
 
         return None
 
-    def _add_repo(self, ksrepo):
+    def _add_repo(self, ksrepo, enabled=True):
```

Comment:
Please remove these changes you have done in the `dnfpayload` file completely. They are too invasive. Instead do the changes only in the `updateBaseRepo` method.

Reply:
With removing the flag for `_add_repo` I end up duplicating a lot of the code in that function.  Additionally my quick tests show the `_add_repo()` inside of `updateBaseRepo` ends up with the repo enabled, rather than disabled.

Not sure what you'd like to see there.  `_add_repo()` is private to `dnfpayload.py`